### PR TITLE
Add more Inspektor Gadget talks

### PR DESCRIPTION
**Pull request by marga-kinvolk**
Recent talks were missing, this adds a few to the list. I'm sure I'm missing some others, but that's what I could find.

### CODE REVIEW

File: README.md
Comment (mauriciovasquezbernal) [inline README.md:93]: This link is wrong, it's the same as above).

**Review by mauriciovasquezbernal — CHANGES_REQUESTED** [review]

File: README.md
Comment (marga-kinvolk) [reply README.md:93]: Oops, you're right, copy & paste error, thanks for catching it!

**Review by mauriciovasquezbernal — COMMENTED** [review]
LGTM now!

**Comment by mauriciovasquezbernal** [comment]
@marga-kinvolk please squash the commits before merging!  :)

**Comment by marga-kinvolk** [comment]
Oops, sorry about the noise, I used an old copy of IG and messed up the rebase. Will redo :-/

**Review by mauriciovasquezbernal — APPROVED** [review]
Explicitly approving to unblock merge.

**Pull request merged**
